main/ui/src/main/resources/css/dark_theme.css

```diff
@@ -105,6 +105,11 @@
 	-fx-font-size: 1.5em;
 }
 
+.label-medium {
```

Comment:
I think we should avoid additional font variations that aren't used anywhere else. @tobihagemann may overrule this :wink:

Reply:
This is something I'd really like to introduce to make user dialogues clearer. This class styles the headings in dialogues, intended to inform the user short and crisp, what the prompt is about. Of course this is not directly releated to function of the PR and can be discussed in a separate issue.

Reply:
I fully agree that the title should get emphasized by using a different size. The point is: Don't we already have enough variations? What about this size:

<img width="270" alt="Screenshot 2020-11-20 at 13 26 43" src="https://user-images.githubusercontent.com/1204330/99800118-2bea4800-2b34-11eb-9964-b05a08635d54.png">

@tobihagemann say something!! 😩

Reply:
I tend to agree that we currently don't have enough variations. `label-large` is just too big for a heading in a dialog. Btw, the button you pointed out has the class `button-large`, which sets the font size to `1.25em`. But `label-large` sets the font size to `1.5em`. We don't have a `label-*` class for `1.25em` (or something like that).

Maaaybe we can set `label-large` in general to `1.25em` but we'd have to check where it's currently used.

Reply:
Before an already existing style class is changed, i'll defer the decision of the size and/or new style class to another issue and set the class for now to `label-large`.

Reply:
Resolved in 63812278974fb7cd8cb2864eaccbc983aa1b6eff

Reply:
Issue about styling is https://github.com/cryptomator/cryptomator/issues/1430